src/System.Private.CoreLib/shared/Interop/Windows/Advapi32/Interop.EventTraceGuidsEx.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        internal enum TRACE_QUERY_INFO_CLASS
+        {
+            TraceGuidQueryList,
+            TraceGuidQueryInfo,
+            TraceGuidQueryProcess,
+            TraceStackTracingInfo,
+            MaxTraceSetInfoClass
+        };
```

Comment:
nit: would be nice to clean up the trailing semicolons

Reply:
Good point. I'll clean that up along the unnecessary marshalling attributes.